src/meltano/cli/schedule.py

```diff
@@ -27,57 +32,154 @@ def schedule(project, ctx):
+    schedule_service: ScheduleService = ctx.obj["schedule_service"]
 
+    _, session_maker = project_engine(project)
+    session = session_maker()
+    try:
+        added_schedule = schedule_service.add(name, job, interval)
         tracker = GoogleAnalyticsTracker(schedule_service.project)
         tracker.track_meltano_schedule("add", added_schedule)
-        click.echo(f"Scheduled '{added_schedule.name}' at {added_schedule.interval}")
-    except ScheduleAlreadyExistsError as serr:
-        click.secho(
-            f"Schedule '{serr.added_schedule.name}' already exists.", fg="yellow"
+        click.echo(
+            f"Scheduled job '{added_schedule.name}' at {added_schedule.interval}"
         )
+    except ScheduleAlreadyExistsError:
+        click.secho(f"Schedule '{name}' already exists.", fg="yellow")
     finally:
         session.close()
 
 
+@schedule.command(short_help="[default] Add a new schedule.")
+@click.argument("name")
+@click.option("--interval", "-i", required=True, help="Interval of the schedule.")
+@click.option("--job", "-j", help="The name of the job to run.")
+@click.option("--extractor", "-e", required=False, help="ELT Only")
+@click.option("--loader", "-l", required=False, help="ELT Only")
+@click.option(
+    "--transform",
+    "-t",
+    type=click.Choice(["skip", "only", "run"]),
+    default="skip",
+    help="ELT Only",
+)
+@click.option("--start-date", type=click.DateTime(), default=None, help="ELT Only")
+@click.pass_context
+def add(ctx, name, job, extractor, loader, transform, interval, start_date):
+    """
+    Add a new schedule. Schedules can be used to run Meltano jobs or ELT tasks at a specific interval.
+
+    Example usage:
+
+    \b
+    \t# Schedule a job name "my_job" to run everyday
+    \tmeltano schedule add <schedule_name> --job my_job --interval "@daily"
+    \t# Schedule an ELT task to run hourly
+    \tmeltano schedule add <schedule_name> --extractor <tap> --loader <target> --transform run --interval "@hourly"
+
+    \b\nNote that the --job option and --extractor/--loader options are mutually exclusive.
+
+    \b\nRead more at https://docs.meltano.com/reference/command-line-interface#schedule
+    """
+    if job and (extractor or loader):
+        raise click.ClickException(
+            "Cannot mix --job with --extractor/--loader/--transform"
+        )
+
+    if not job:
+        if not extractor:
+            raise click.ClickException("Missing --extractor")
+        if not loader:
+            raise click.ClickException("Missing --loader")
+
+        _add_elt(ctx, name, extractor, loader, transform, interval, start_date)
+        return
+    _add_job(ctx, name, job, interval)
+
+
+def _format_job_list_output(entry: Schedule, job: TaskSets) -> dict:
+    return {
+        "name": entry.name,
+        "interval": entry.interval,
+        "cron_interval": entry.cron_interval,
+        "env": entry.env,
+        "job": {
+            "name": job.name,
+            "tasks": job.tasks,
```

Comment:
Since files-airflow will need it (because we're going to do a dag per job task), I fetch the job tasks and display them. We could skip this and just force files-airflow to call `meltano jobs list` to get a list of job/tasks.

Reply:
@pandemicsyn - Thanks for calling this out. Both ways could work, but I think it's completely reasonable to include `tasks` here, as you have it - and I think that'd be my slight preference. Although it's redundant with what one would get from `meltano list jobs`, there's clear value in including it. And there's benefit in not needing to make two calls when one will suffice.

Certain applications will need to ignore it, like the proposed `--format=crontab` and/or a cron orchestrator which won't be able to invoke the tasks separately anyway. However, it seems reasonable to include this on default outputs, and callers can always ignore the `tasks` entry if they want to invoke directly by job name.